tests/test_cli.py

```diff
@@ -1,9 +1,12 @@
-import subprocess
-import sys
+from fastcs.launch import _extract_options_model
+from ruamel.yaml import YAML
 
-from thorlabs_mff_fastcs import __version__
+from thorlabs_mff_fastcs.controllers import ThorlabsMFF
 
 
-def test_cli_version():
-    cmd = [sys.executable, "-m", "thorlabs_mff_fastcs", "--version"]
-    assert subprocess.check_output(cmd).decode().strip() == __version__
+# Add test for schema changes (assess version changes)
+def test_schema(data):
+    ref_schema = YAML(typ="safe").load(data / "schema.json")
+    options_model = _extract_options_model(ThorlabsMFF)
+    target_schema = options_model.model_json_schema()
+    assert ref_schema == target_schema
```

Comment:
This looks good. You are probably going to want a convenient way to regenerate the checked in schema, like a vscode task, or a script. pvi does the latter https://github.com/epics-containers/pvi/blob/main/tests/regenerate_test_output.sh